Terraform/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    bucket = "8586-terraform-state"
+    key    = "sandesh/terraform.tfstate"
+    region = "us-east-1"
```

Comment:
lets use github actions rather than using hardcoded

Reply:
Ok done. Passing the values using -backend-config option.